protocol/flatbuffers/protocol.fbs

```diff
@@ -118,7 +118,8 @@ table Transaction {
   sampling_priority:      double;  // added in the 8.2 PHP agent release
   span_events:            [Event];
   log_events:             [Event]; // added in the 10.1 PHP agent release
-  php_packages:           Event;   // added in the ??? PHP agent release
+  php_packages:           Event;   // added in the 10.17 PHP agent release
+  log_forwarding_labels:  Event;   // added in the ??? PHP agent release
```

Comment:
I think you can go ahead and put 11.7 in there instead of ???

Reply:
Handled here https://github.com/newrelic/newrelic-php-agent/pull/1027/commits/467c8a2f1bde998311d2633ba374edc5cddf58a8